### PR TITLE
Fix/header popup

### PR DESCRIPTION
**Pull request by CzarekDryl**
Fixes to header components

### CODE REVIEW

File: src/components/common/Extensions/ColonySwitcher/partials/ColonyAvatarWrapper.tsx [hunk 1]
```diff
@@ -16,38 +16,47 @@ const ColonyAvatarWrapper: FC<ColonyAvatarProps> = ({
...
-      })}
-      ref={setTriggerRef}
-    >
-      <Icon name="caret-up" appearance={{ size: 'extraTiny' }} />
-    </span>
-  </>
-);
+
+      {isMobile && colonyToDisplay && (
+        <div className="font-semibold text-md text-gray-900 mx-2">
```
Comment (KatarzynaStefanowiczPagepro) [inline src/components/common/Extensions/ColonySwitcher/partials/ColonyAvatarWrapper.tsx:38]: Do we need text-gray-900?

File: src/components/frame/Extensions/Header/Header.tsx [hunk 2]
```diff
@@ -154,19 +154,21 @@ const Header = () => {
...
-              </p>
-            </button>
+            {!visible && (
+              <button
+                type="button"
+                className={clsx('items-center flex sm:hidden', {
+                  'opacity-100 visible': !isMainMenuVisible,
+                  'opacity-0 invisible': isMainMenuVisible,
+                })}
+                ref={mainMenuSetTriggerRef}
```
Comment (KatarzynaStefanowiczPagepro) [inline src/components/frame/Extensions/Header/Header.tsx:164]: Add aria-label here

Comment (CzarekDryl) [reply src/components/frame/Extensions/Header/Header.tsx:158]: aria-label added

File: src/components/common/Extensions/ColonySwitcher/partials/ColonyAvatarWrapper.tsx (hunk 1 above)
Comment (CzarekDryl) [reply src/components/common/Extensions/ColonySwitcher/partials/ColonyAvatarWrapper.tsx]: Removed

**Review by KatarzynaStefanowiczPagepro — APPROVED** [review]

File: src/components/frame/Extensions/Header/Header.tsx [hunk 3]
```diff
@@ -62,21 +62,15 @@ const Header = () => {
...
                 type="button"
               >
                 <ColonyAvatarWrapper
-                  isOpen={visible}
+                  isOpen={visible || isMainMenuVisible}
                   isMobile={isMobile}
                   colonyToDisplayAddress={colonyToDisplayAddress}
                   colonyToDisplay={
-                    isMainMenuVisible ? colonyToDisplay : undefined
+                    isCloseButtonVisible ? colonyToDisplay : undefined
```
Comment (krawczykkatarzyna) [inline src/components/frame/Extensions/Header/Header.tsx:73]: can't be just `isCloseButtonVisible && colonyToDisplay` ?

**Pull request merged**